flink-mesos/build.gradle.kts

```diff
@@ -3,21 +3,21 @@ plugins {
 }
 
 dependencies {
-    implementation("org.apache.mesos:mesos:1.0.1")
-    implementation("com.netflix.fenzo:fenzo-core:0.10.1")
-    implementation("org.apache.flink:flink-shaded-netty:4.1.32.Final-7.0")
-    testImplementation("org.scalatest:scalatest_${Versions.baseScala}:3.0.0")
-    testImplementation("org.apache.curator:curator-test:2.12.0")
-    testImplementation("com.typesafe.akka:akka-testkit_${Versions.baseScala}:2.5.21")
+    implementation(Libs.mesos)
+    implementation(Libs.fenzo_core)
+    implementation(Libs.flink_shaded_netty)
+    testImplementation(Libs.scalatest_2_11)
```

Comment:
I think we still need to do some cleanups, for example use `Versions.baseScala` in the Scala dependencies so we don't have the explicit `2_11` here.

Reply:
Yes, but much later. First get it to compile. Then make it more beautiful.

Reply:
👌